tests/Microsoft.Identity.Test.Unit.net45/ApiConfigTests/AcquireTokenInteractiveBuilderTests.cs

```diff
@@ -147,14 +147,54 @@ public async Task TestAcquireTokenInteractive_EmbeddedAndSystemOptions_Async()
             var options = new SystemWebViewOptions();
             var ex = await AssertException.TaskThrowsAsync<MsalClientException>(() =>
 
-                 AcquireTokenInteractiveParameterBuilder.Create(_harness.Executor, MsalTestConstants.Scope)
+                 AcquireTokenInteractiveParameterBuilder.Create(_harness.Executor, null, MsalTestConstants.Scope)
                                                          .WithSystemWebViewOptions(options)
                                                          .WithUseEmbeddedWebView(true)
                                                          .ExecuteAsync()
                                                         ).ConfigureAwait(false);
 
             Assert.AreEqual(MsalError.SystemWebviewOptionsNotApplicable, ex.ErrorCode);
         }
+
+        [TestMethod]
```

Comment:
>[TestMethod] [](start = 8, length = 12)

Seems like some tests for the setting the activity call back on the PCA builder is missing?